Source/Private/_ASPendingState.mm

```diff
@@ -150,6 +143,24 @@ @implementation _ASPendingState
   NSDictionary<NSString *, id<CAAction>> *actions;
 
   ASPendingStateFlags _flags;
```

Comment:
Maybe we could rename this to something else and name _boolFlags _flags? Maybe this should be _stateToApplyFlags?

Reply:
Good idea.